plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeAnalyze.java

```diff
@@ -16,7 +16,7 @@
 import com.google.common.collect.ImmutableMap;
 import io.trino.Session;
 import io.trino.operator.OperatorStats;
-import io.trino.plugin.deltalake.util.DockerizedMinioDataLake;
+import io.trino.plugin.hive.containers.HiveMinioDataLake;
```

Comment:
This test class looks broken now. I can't run locally after removing `abstract`. 
```
java.lang.IllegalStateException: Mapped port can only be obtained after the container is started
```

Reply:
it was missing `hiveMinioDataLake.start()`. Succesfully tested locally.

Reply:
It works now, thanks!